autogen/img_utils.py

```diff
@@ -298,3 +310,51 @@ def message_formatter_pil_to_b64(messages: List[Dict]) -> List[Dict]:
         new_messages.append(message)
 
     return new_messages
+
+
+def is_vision_model(model_name: str) -> bool:
```

Comment:
Do we need this function if the `vision-model` parameter exists in the LLM config? Or is this purely for backward compatibility?

Reply:
Very good question. I think for backward compatibility -- relying on the name is not a winning strategy. 

E.g., on our Azure environment, the vision model is named "_gpt-4-turbo-v_" which would fail the regex.

Reply:
@afourney  @WaelKarkoub  That's why we have an "or" operation in the `client.py` file. Even if it is not identified as a vision model from the model's name, passing a parameter `"vision_model": true` would override this issue.